nautobot/dcim/tests/test_filters.py

```diff
@@ -1505,302 +1520,420 @@ def setUpTestData(cls):
         )
 
     def test_model(self):
-        params = {"model": ["Model 1", "Model 2"]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        models = ["Model 1", "Model 2"]
+        params = {"model": models}
+        self.assertEqual(self.filterset(params, self.queryset).qs.count(), len(models))
 
     def test_slug(self):
-        params = {"slug": ["model-1", "model-2"]}
-        self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        slugs = ["model-1", "model-2"]
```

Comment:
```suggestion
        slugs = DeviceType.objects.values_list("slug", flat=True).distinct()[:2]
```

Reply:
I was really trying not to muck with these right now since they are all relying on this gross hard-coding and because of the factory-generated fixtures, the slicing doesn't work to successfully pass tests consistently. :(